stdlib/public/core/Hashing.swift

```diff
@@ -147,51 +147,28 @@ func _mixInt(_ value: Int) -> Int {
 #endif
 }
 
-/// Given a hash value, returns an integer value within the given range that
-/// corresponds to a hash value.
+/// Given a hash value, returns an integer value between 0 and `upperBound`
```

Comment:
To be precise, can we say something like "an integer value in the range 0..<upperBound", as it is not clear to me from this description whether `upperBound` is included or not.